src/events.cpp

```diff
@@ -133,6 +133,12 @@ bool Events::load()
 			} else {
 				std::cout << "[Warning - Events::load] Unknown monster method: " << methodName << std::endl;
 			}
+		} else if (className == "Game") {
+			if (methodName == "onLuaApiResponse") {
+				info.MiscOnLuaApiResponse = scriptInterface.getEvent(methodName);
```

Comment:
Misc -> misc

Reply:
I completly forgot to convert this, thanks for pointing it out